draft-ietf-taps-interface.md

```diff
@@ -801,6 +801,15 @@ The Listen() Action consumes the Preconnection. Once Listen() has been
 called, no further parameters may be bound to the Preconnection, and no
 subsequent establishment call may be made on the Preconnection.
 
+Listening continues until the global context shuts down, or until the Stop
+action is performed on the same Preconnection:
+
+~~~
+Preconnection.Stop()
+~~~
+
+After Stop() is called, the preconnection can be disposed of.
```

Comment:
What's the point of this - how is it different from Abort?

Reply:
Stop() is ordely. Abort() is more abrupt (at least, that should be the intent...)

Reply:
Ah, sorry, missed this, it's for cleaning up a Preconnection. That makes sense to me. But semantically this seems quite similar to abort to me - a way to say "I don't want to use this anymore, from now on" - so why not give it the same name?  (though I don't care much about the name)

Reply:
Is there a reason not calling it "Preconnection.Abort" ?

Reply:
Exactly what I meant - sorry for not saying it clearer

Reply:
IMO `Abort()` should be used for exceptional or abnormal termination, and this is used for a normal termination of listening.